pkg/customflag/validate.go

```diff
@@ -44,100 +42,121 @@ func ValidateTemplateFlags() {
 		validateMultusTest(expected, expectedUpgrade, valuesUpgrade)
 	case "components":
 		validateComponentsTest(expected, expectedUpgrade, valuesUpgrade)
+	case "flannel":
+		validateFlannelTest()
 	default:
-		shared.LogLevel("error", "test tag not found")
+		log.Errorf("test tag not found")
 	}
 }
 
 func validateVersionBumpTest(expectedValue, expectedUpgrade []string, valuesUpgrade string) {
 	cmd := os.Getenv("CMD")
 	if cmd == "" {
-		shared.LogLevel("error", "cmd was not sent")
+		log.Errorf("cmd was not sent")
 		os.Exit(1)
 	}
 
 	cmdLenght := strings.Split(cmd, ",")
 	if len(cmdLenght) != len(expectedValue) {
-		shared.LogLevel("error", "mismatched length commands: %d x expected values: %d",
-			len(cmdLenght), len(expectedValue))
+		log.Errorf("mismatched length commands: %d x expected values: %d", len(cmdLenght), len(expectedValue))
 		os.Exit(1)
 	}
 
 	if valuesUpgrade != "" {
 		if len(expectedUpgrade) != len(expectedValue) {
-			shared.LogLevel("error", "mismatched length commands: %d x expected values upgrade: %d",
-				len(cmdLenght), len(expectedValue))
+			log.Errorf("mismatched length commands: %d x expected values upgrade: %d",
+				len(expectedUpgrade), len(expectedValue))
 			os.Exit(1)
 		}
 	}
 }
 
+func validateFlannelTest() {
+	cmd := os.Getenv("CMD")
+	if cmd != "" {
+		log.Errorf("cmd can not be sent for flannel tests as it is already defined in the test file")
+		os.Exit(1)
+	}
+}
+
 func validateCiliumTest(expectedValue, valuesUpgrade []string, upgrade string) {
-	ciliumCmdsLenght := 2
+	cmd := os.Getenv("CMD")
+	if cmd != "" {
+		log.Errorf("cmd can not be sent for cilium tests as it is already defined in the test file")
+		os.Exit(1)
+	}
 
+	ciliumCmdsLenght := 2
```

Comment:
typo: `ciliumCmdsLength`

Reply:
thats odd, i swear that we had that English typos check on ci